mm/2s2h/BenGui/QuickStart.cpp

```diff
@@ -0,0 +1,470 @@
+    float lineY = textPos.y + textSize.y * 0.5f;
+    ImU32 lineColor = IM_COL32(255, 255, 255, 255);
+
+    drawList->AddLine(ImVec2(region.x, lineY), ImVec2(textPos.x - spacing, lineY), lineColor, 2.0f);
+    drawList->AddLine(ImVec2(textPos.x + textSize.x + spacing, lineY), ImVec2(region.y, lineY), lineColor, 2.0f);
+
+    ImGui::SetCursorScreenPos(ImVec2(textPos.x, textPos.y));
+    ImGui::TextColored(TEXT_COLOR(COLOR_WHITE), text);
+}
+
+void SetPresetOption(QuickStartSettings option) {
+    if (option != QUICK_START_NONE) {
+        PresetManager_ApplyPreset(option == QUICK_START_PRESET_DEFAULT ? defaultsPresetJ : vanillaEnhancedPresetJ);
+        Ship::Context::GetInstance()->GetWindow()->GetGui()->SaveConsoleVariablesNextFrame();
+    }
+}
+
+void SetRandoQuickStartOptions() {
+    Ship::Context::GetInstance()->GetConsoleVariables()->ClearBlock("gRando.Options");
+
+    // Logic
+    CVarSetInteger(Rando::StaticData::Options[RO_LOGIC].cvar,
+                   quickStartOptions.rando.logicOption == QUICK_START_RANDO_GLITCHLESS ? RO_LOGIC_GLITCHLESS
+                                                                                       : RO_LOGIC_NO_LOGIC);
+
+    // Hints
+    CVarSetInteger(Rando::StaticData::Options[RO_HINTS_GOSSIP_STONES].cvar, 1);
+    CVarSetInteger(Rando::StaticData::Options[RO_HINTS_BOSS_REMAINS].cvar, 1);
+    CVarSetInteger(Rando::StaticData::Options[RO_HINTS_OATH_TO_ORDER].cvar, 1);
+    CVarSetInteger(Rando::StaticData::Options[RO_HINTS_HOOKSHOT].cvar, 1);
+
+    // Base Game Shuffles
+    CVarSetInteger(Rando::StaticData::Options[RO_SHUFFLE_OWL_STATUES].cvar, 1);
+    CVarSetInteger(Rando::StaticData::Options[RO_SHUFFLE_SHOPS].cvar, 1);
+    CVarSetInteger(Rando::StaticData::Options[RO_SHUFFLE_TINGLE_SHOPS].cvar, 1);
+    CVarSetInteger(Rando::StaticData::Options[RO_SHUFFLE_GOLD_SKULLTULAS].cvar, 1);
+    CVarSetInteger(Rando::StaticData::Options[RO_MINIMUM_STRAY_FAIRIES].cvar, 15);
+    CVarSetInteger(Rando::StaticData::Options[RO_SHUFFLE_BOSS_REMAINS].cvar, 1);
+
+    if (quickStartOptions.rando.shuffleSet == QUICK_START_RANDO_ALL) {
+        CVarSetInteger(Rando::StaticData::Options[RO_SHUFFLE_POT_DROPS].cvar, 1);
+        CVarSetInteger(Rando::StaticData::Options[RO_SHUFFLE_CRATE_DROPS].cvar, 1);
+        CVarSetInteger(Rando::StaticData::Options[RO_SHUFFLE_BARREL_DROPS].cvar, 1);
+        CVarSetInteger(Rando::StaticData::Options[RO_SHUFFLE_SNOWBALL_DROPS].cvar, 1);
+        CVarSetInteger(Rando::StaticData::Options[RO_SHUFFLE_GRASS_DROPS].cvar, 1);
+        CVarSetInteger(Rando::StaticData::Options[RO_SHUFFLE_FREESTANDING_ITEMS].cvar, 1);
+        CVarSetInteger(Rando::StaticData::Options[RO_SHUFFLE_BOSS_SOULS].cvar, 1);
+    }
+
+    Ship::Context::GetInstance()->GetWindow()->GetGui()->SaveConsoleVariablesNextFrame();
+}
+
+void QuickStartQuestInit(const char* fileNameEntry) {
+    static std::string cleanFileName = Ship_RemoveSpecialCharacters(fileNameEntry);
```

Comment:
The `static` keyword means that any subsequent quickstart saves I make will always have the same name in the same runtime.
```suggestion
    std::string cleanFileName = Ship_RemoveSpecialCharacters(fileNameEntry);
```